src/mono/wasm/Wasm.Build.Tests/TestAppScenarios/MemoryTests.cs

```diff
@@ -37,4 +38,25 @@ public async Task AllocateLargeHeapThenRepeatedlyInterop()
             await RunSdkStyleAppForBuild(new (Configuration: config, TestScenario: "AllocateLargeHeapThenInterop"));
         }
     }
+
+    [Fact]
+    public async Task RunSimpleAppWithProfiler()
+    {
+        string config = "Release";
+        CopyTestAsset("WasmBasicTestApp", "ProfilerTest", "App");
+        string extraArgs = "-p:WasmProfilers=log; -p:WasmBuildNative=true";
```

Comment:
let's enable all 3 profilers here I think it should link just fine. 
And then we only initialize one of them from the JS side.

Reply:
You mean: 3 separate tests?

Reply:
Possibly, but testing just `log` for now would be OK. 
I really mean just to test linking all of them into native wasm.